src/components/selections/defaults.js

```diff
@@ -49,6 +49,8 @@ function handleSelectionDefaults(selectionIn, selectionOut, fullLayout) {
     coerce('line.color');
     coerce('line.width');
     coerce('line.dash');
+    coerce('xshift');
```

Comment:
Please check the axis `type` before coercing.

Reply:
done 👍